.gitlab-ci.yml

```diff
@@ -96,7 +96,8 @@ build:mender-client-acceptance-testing:
   stage: build
   needs: []
   tags:
-    - mender-qa-slave-highcpu
+    # KVM support requires the same type of GCP machine as the tests will use
```

Comment:
Why?

Reply:
I learn this the hard way, but it is because we install Linux kernel modules based on the real machine kernel version. See [here](https://github.com/mendersoftware/mender-test-containers/blob/58d64695c2efa2b624d67068c7fe5e84b731400c/mender-client-acceptance-testing/Dockerfile#L14).